timemachine/cpp/src/local_md_config.cu

```diff
@@ -0,0 +1,197 @@
+#include "constants.hpp"
+#include "gpu_utils.cuh"
+#include "kernel_utils.cuh"
+#include "kernels/k_flat_bottom_bond.cuh"
+#include "kernels/k_indices.cuh"
+#include "kernels/k_local_md.cuh"
+#include "local_md_config.hpp"
+#include "math_utils.cuh"
+#include <cub/cub.cuh>
+#include <random>
+#include <vector>
+
+namespace timemachine {
+
+// Struct to as a CUB <= operation
+struct LessThan {
+    int compare;
+    CUB_RUNTIME_FUNCTION __device__ __forceinline__ explicit LessThan(int compare) : compare(compare) {}
+    CUB_RUNTIME_FUNCTION __device__ __forceinline__ bool operator()(const int &a) const { return (a < compare); }
+};
+
+LocalMDConfig::LocalMDConfig(const int N, const std::vector<std::shared_ptr<BoundPotential>> bps)
```

Comment:
naming nit: to me `LocalMDConfig` suggests an object that holds configuration values, but this seems like more of a wrapper for instantiated potentials? Don't immediately have a good suggestion, but maybe something like `LocalMDPotentials` would be preferable?